### PR TITLE
add SetSearchLive,SetSearchPhotos, and SetSearchVideos

### PR DESCRIPTION
**Pull request by JustHumanz**
add realtime searching and filtering tweets like photos/videos only

### CODE REVIEW

**Comment by n0madic** [comment]
Name of methods that change properties will confuse with SearchTweets...
From the comments can see that there was originally a `Set` prefix, I think it's worth returning it

**Comment by JustHumanz** [comment]
hemm make sense,wait a sec

**Review by n0madic — CHANGES_REQUESTED** [review]
ok?

File: search.go [hunk 1]
```diff
@@ -37,6 +37,12 @@ func (s *Scraper) FetchSearchTweets(query string, maxTweetsNbr int, cursor strin
 	if cursor != "" {
 		q.Add("cursor", cursor)
 	}
+	if s.searchMode == "live" {
```
[the remark this answers is not in the document]
Comment (JustHumanz) [reply search.go:40]: ah right,i never thinking about that

**Comment by JustHumanz** [comment]
sure

**Review by JustHumanz — COMMENTED** [review]
lgtm

**Comment by n0madic** [comment]
no more changes?

**Review by JustHumanz — COMMENTED** [review]
yeah, this looks good

**Comment by n0madic** [comment]
[i meant it](https://github.com/n0madic/twitter-scraper/commit/922ccc5a874216834331ef8d5bf3c627225c1081)

**Comment by n0madic** [comment]
[Changed mode selection to enum](https://github.com/n0madic/twitter-scraper/commit/1d3f84fc4ac93da53c7d0899bf7bef5b1d9c1712)

**Pull request merged**